editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -4957,6 +5013,7 @@ void Node3DEditor::update_transform_gizmo() {
 	gizmo.visible = count > 0;
 	gizmo.transform.origin = (count > 0) ? gizmo_center / count : Vector3();
 	gizmo.transform.basis = (count == 1) ? gizmo_basis : Basis();
+	gizmo.target_center = gizmo_center;
```

Comment:
Is this meant to be `gizmo_center / count`? How does the new mode handle multi-node selection?

Reply:
Good catch, it didn't handle that before, fixed.